src/main.rs

```diff
@@ -167,15 +190,7 @@ async fn main() {
     }
 
     if !args.cli {
-        let mut builder = pretty_env_logger::formatted_timed_builder();
-
-        builder
-            .parse_filters(
-                env::var(env_logger::DEFAULT_FILTER_ENV)
-                    .unwrap_or_else(|_| "sqlx=warn,info".to_string())
```

Comment:
think there was a reason for this sqlx default (it's too spammy by default or vice versa, don't remember)

Reply:
Yeah seems like it was to spammy: 398b9947b06878d67df1bb38272db740225fd698

However I'm not seeing this anymore (nothing appears by default on INFO level).